storage/oasis/nodeapi/damask/node.go

```diff
@@ -75,7 +77,13 @@ func (c *DamaskConsensusApiLite) GetTransactionsWithResults(ctx context.Context,
 	for i, txBytes := range rsp.Transactions {
 		var tx consensusTx.SignedTransaction
 		if err := cbor.Unmarshal(txBytes, &tx); err != nil {
-			return nil, fmt.Errorf("GetTransactionsWithResults(%d): transaction %d (%x): %w", height, i, txBytes, err)
+			log.NewDefaultLogger("damask-consensus-api-lite").Error("malformed consensus transaction, leaving empty",
```

Comment:
I guess we donćt have to do the same for Cobalt because it's set in stone, and we know that there were no malformed blocks?

Reply:
let's add it there too